GUI/functions.py

```diff
@@ -19,3 +19,12 @@ def in_database(link):
 		print("No game with that name found")
 	elif link == "Doesn’t support":
 		print("Game doesn't support this OS")
+
+def checkversion(version1, version2):
```

Comment:
This is going to break for a lot of games. Some games use dates as version (jan/10/2017, 7/11/2018) others use month names (December release) and some even use code names. You should add a check to make sure the game uses the standard . system (1.0.0 ect)